### DISCUSSION TITLE
from pyproj._network import (  # noqa: F401 pylint: disable=unused-import  ImportError: DLL load failed while importing _network: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by anwarmaxud** [pyproj4/pyproj#1370] [General]
Hello experts,
I am getting this error while trying to run the package/tool in Spyder. The pyproj version is 3.6.0, gdal = 3.6.2 and python 3.11.4. Any ideas why this is happening

**Comment by djhoese** [comment]
Please provide information about your system, how you installed pyproj, and the complete traceback of the import error. My best guess is that pyproj was not installed properly. Pyproj includes C extensions that must be compiled when installing from the source code or you must install a binary wheel compatible with your system where the extensions are pre-built. If you used `pip` to install pyproj then by default it should have tried to find a pre-built wheel compatible with your system.

**Reply by anwarmaxud** [reply]
Hi, thanks for writing back. The issue has been resolved. I made three changes.
1- I updated the Pyproj package through the pip installer.
2- Added a PROJ_LIB environment variable 
3- Previously, I had a separate installation of GDAL to which the GDAL_DATA environment variable was connected to. I changed the address to the one located in Anaconda\Library installation
